### PR TITLE
Add Justpaste.it translator

### PR DESCRIPTION
**Pull request by febrezo**
It stores the website as an attachment. Test case included.

### CODE REVIEW

**Comment by adam3smith** [comment]
All comments on https://github.com/zotero/translators/pull/1131/files apply here, too

File: Justpaste.it.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
+{
+	"translatorID": "1d1a51d4-60bf-44b8-ae28-8b154d1ed721",
+	"label": "Justpaste.it",
```
Comment (zuphilip) [inline Justpaste.it.js]: This is the same ID as you used in #1131, but the same ID cannot be used for two different translators.

File: Justpaste.it.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+	//Setting common data:
+	newItem.websiteTitle = "Justpaste.it";
+	newItem.url = url;
+	var auxTitle = ZU.xpathText(doc, '//title');	
+	newItem.title = auxTitle.substring(0, auxTitle.lastIndexOf(" - justpaste.it"));
+	newItem.websiteType = "Paste Site";
+
+	var auxDate = ZU.xpathText(doc, '//div[@class="col-md-3 col-xs-4 noteBottomFooter"]');
+	if (auxDate.indexOf("ago")>=0) {
```
Comment (adam3smith) [inline Justpaste.it.js]: There are several problems here:
1. The also counts days, so will get the wrong date (e.g. for you test). You could read up on how you can go x days back in a javascript data items -- I'm sure there's something on stackoverflow, I don't know ottomh.
2. months in js date items start at 0, so you need to add 1
3. you get the day using `.getDate()`, not `Day`.

**Comment by AbeJellinek** [comment]
@febrezo, would you be able to rebase on `master`, lint, and make the requested changes?

**Comment by AbeJellinek** [comment]
Closing for lack of activity.

**Pull request closed without merging**